src/metadata/bpm.cpp

```diff
@@ -0,0 +1,60 @@
+#include "metadata/bpm.h"
```

Comment:
I will open a separate PR for the TagLib and metadata related changes once 
the PlayCounter PR has been merged. This is a good opportunity for moving 
all the track-related stuff into the corresponding folder.

On 12/27/2015 05:51 PM, RJ Ryan wrote:
>
> In src/metadata/bpm.cpp 
> <https://github.com/mixxxdj/mixxx/pull/728#discussion_r48456431>:
>
> > @@ -0,0 +1,60 @@
> > +#include "metadata/bpm.h"
>
> We already have a folder for track related objects -- "track". Could you 
> move this in there instead of 'metadata'?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/728/files#r48456431>.
>